### ISSUE TITLE
Giphy support

### ISSUE THREAD

**Issue by AkbarTheGreat** [AkbarTheGreat/cszb-scoreboard#7]
Similar to https://github.com/AkbarTheGreat/cszb-scoreboard/issues/6, but for Giphy. I don't know if wxDC supports animated gifs by default, so this may require implementing an animated gif animator.

**Comment by hsq-cernansky** [comment]
I'm guessing that wxDC supports them. They're ubiquitous anymore. I learned .NET because originally VB6 didn't support animated GIFs, and I wanted JANIS to use them.

Giphy, though, isn't really gifs, right? It's a little video stream, unless I'm completely mistaken.

**Comment by AkbarTheGreat** [comment]
You're probably correct about Giphy being HTML5 video streams now instead of gifs (although since their branding is so heavily "gif" I kind of assumed they also can serve a gif of whatever?)

In any case, wxDC by default appears to just render one frame of an animated gif from my early testing.  I kind of see all of these "display animation" tickets as being closely related.

That said, gif support and Giphy support may be related, but different tasks.

**Comment by BrBill** [comment]
Yeah, I get it. Maybe there's a setting in wxDC, or maybe somebody out there has shared out a way to do it that can be adopted here.

I forgot and connected via my work account above! Sorry about that, I'll try to remember to use my personal acct.
